Lecture6(GithubAPI)/Component/BodyComponent.js

```diff
@@ -26,14 +28,13 @@ const BodyComponent = () => {
     useEffect(() => {
         if (!sessionStorage.getItem('teamData')) {
             getGithubApiData().then((githubdata) => {
```

Comment:
Are you trying data from session storage and api if not available in session storage? 
session storages expires as soon as a tab is closed.
Also what if you get an error from the api you are fetching data from?

Reply:
I would suggest local storage for that. Also now you can handle api cases.

Reply:
Yes will implement error handling